ops/dev/test-pacs-restore.ps1

```
@@ -0,0 +1,69 @@
+  1) Docker engine is reachable
+  2) SQL Server container is running
+  3) pacs_oltp database exists
+  4) Trivial query returns data
+
+  Usage:
+    pwsh -NoProfile -File ops/dev/test-pacs-restore.ps1 -SaPassword 'YOUR_PASSWORD'
+#>
+
+param(
+  [Parameter(Mandatory=$true)]
+  [string]$SaPassword,
```

PSScriptAnalyzer (1.24.0)

[warning] Parameter '$SaPassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $SaPassword type to SecureString; Set $SaPassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)